.github/actions/next-repo-actions/src/triage-issues-with-ai.ts

```diff
@@ -1,7 +1,10 @@
 import { info } from '@actions/core'
+import { context } from '@actions/github'
 
 async function main() {
-  info('Triaging issues with AI!')
+  const issue = context.payload.issue
+  info(`Issue: ${issue}`)
+  info(`Issue: ${JSON.stringify(issue)}`)
```

Comment:
Do you wanna pretty print this?
```suggestion
  info(`Issue: ${JSON.stringify(issue, null, 2)}`)
```

Reply:
@ijjk <picture data-single-emoji=":til:" title=":til:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/til/d5952483dda1dda3.png" alt=":til:" align="absmiddle"></picture>